README.md

```diff
@@ -151,6 +169,8 @@ Method             | Description
 `bar(int)`         | Creates a horizontal divider bar of the given length
 `create(String)`   | Creates a new JSONMessage with the given text as a starting point
 `color(ChatColor)` | Sets the color of the current message part
+`color(String)`    | Same as `color(ChatColor)` but allows the usage of HEX colors in 1.16 and newer
```

Comment:
hmmmm. should we mention here that they can pass in a default ChatColor to use if it's not on 1.16? i feel like at this point actual javadocs should be used to generate a docs site...

Reply:
Seems like an idea.

Reply:
hmm. i think it's fine how it is for now, there are javadocs in the code which can be used anyways if people really want, and I want to keep these docs simple. eventually I'll move to actual published javadocs at some point :crossed_fingers: